src/components/Home/index.js

```diff
@@ -0,0 +1,40 @@
+import { Box, Flex, Image, Container, Spacer, Text } from "@chakra-ui/react";
+
+export default function Home() {
+  return (
+    <Flex color="white" bg={"#D9753B"} height={"100vh"}>
```

Comment:
Try to use 'theme' for the colors

https://chakra-ui.com/docs/styled-system/theming/customize-theme